### PR TITLE
Update the content of /ceph to match the copy doc

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Update ceph main page

## QA

- Open the website https://ubuntu-com-10400.demos.haus/ceph
- Make sure that the changes matches the copy doc (link in the linked issue)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4423

### CODE REVIEW

**Comment by goulinkh** [comment]
Waiting for design team to check that the new diagram is good

File: templates/ceph/index.html [hunk 1]
```diff
@@ -191,6 +191,7 @@ <h3 class="p-heading--four">Ceph support on Ubuntu</h3>
         <li class="p-list__item is-ticked">Get access to the storage experts 24x7 or on business hours</li>
         <li class="p-list__item is-ticked">Alternatively, get extended security maintenance if all you need is peace of mind by consuming security patches from the world's leading security team</li>
       </ul>
+      <p><a class="p-link--external" href="/advantage">Find out more</a></p>
```
Comment (mtruj013) [inline templates/ceph/index.html:194]: This isn't an external link so you can remove the `p-link--external` class

File: templates/ceph/index.html [hunk 2]
```diff
@@ -202,6 +203,7 @@ <h3 class="p-heading--four">Charmed Ceph</h3>
         <li class="p-list__item is-ticked">Replace failing disk drives and retire old hardware</li>
         <li class="p-list__item is-ticked">Expand the capacity of your storage cluster</li>
       </ul>
+      <p><a class="p-link--external" href="/ceph/install">Install Charmed Ceph</a></p>
```
Comment (mtruj013) [inline templates/ceph/index.html:206]: Remove `p-link--external` class

File: templates/ceph/index.html [hunk 3]
```diff
@@ -213,12 +215,9 @@ <h3 class="p-heading--four">Managed Ceph</h3>
         <li class="p-list__item is-ticked">Modern monitoring and log management</li>
         <li class="p-list__item is-ticked">We will hand over the keys when you are ready</li>
       </ul>
+      <p><a class="p-link--external" href="/ceph/managed">Find out more</a></p>
```
Comment (mtruj013) [inline templates/ceph/index.html:218]: Remove `p-link--external` class

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Hi @goulinkh! Looks good, just a couple of small changes:

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Comment by goulinkh** [comment]
@bethcollins92 I think the design team already approved the illustration: https://github.com/canonical-web-and-design/web-squad/issues/4424

**Comment by bethcollins92** [comment]
Nope - all is good, I just popped it on there as I saw your comment about waiting for design

**Pull request merged**